change/@ni-nimble-components-5084859c-b0ef-4431-b684-b4733c6d7714.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm not entirely convinced what the desired behavior would be for this case. A client might assume that setting an item as disabled would prevent the user from setting that as the value. Do we want to say that assumption is invalid? If so then what's the point of setting an item as disabled?

Some options I can think of:
1. Document that clients shouldn't set combobox items as disabled. Then we can choose whatever behavior we want if clients do this (within the bounds of good custom element behavior, ie we can't just crash). Maybe we'd decide to leave the behavior as-is or maybe we'd ignore the disabled attribute.
2. Document that the value can never be set to the value of a disabled item. Then we'd need to brainstorm the desired UX if the user types the value. While I agree the behavior reported in #2252 is clunky, it does achieve the goal of disallowing that value interactively.
3. Decide that disabled items are more of a suggestion to users, but there are valid use cases for allowing those values to be entered manually. Then this PR is going in the right direction.

It's probably worth a team / PO discussion to try to agree on the direction. @m-akinc

Reply:
From the discussion, it seems like we're going with option 3. I filed #2302 for us to document usage patterns around disabled combobox options. We still want the control to behave sanely when a disabled option value is typed in, and that's what this PR aims to accomplish.

Reply:
Could you update the PR description to include a short summary of the expected behavior from the user's point of view?